schemas/zowe-yaml-schema.json

```diff
@@ -423,6 +423,35 @@
           "additionalProperties": false,
           "description": "Optional, advanced network configuration parameters",
           "properties": {
+            "bind": {
+              "type": "array",
+              "description": "The IP addresses which all of the Zowe servers will be binding to.",
+              "items": {
+                "type": "string"
```

Comment:
dont make it a string. cant do good validation on that.
use this: https://github.com/zowe/zowe-install-packaging/blob/v2.x/staging/schemas/server-common.json#L78

and we should have an ipv6 one too, even though we dont support ipv6 yet...